hack/generate-flavors.sh

```diff
@@ -4,17 +4,25 @@ set -euo pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 FLAVORS_DIR="${REPO_ROOT}/templates/flavors"
+SUPPORTED_DISTROS=(
+    "clusterclass-kubeadm"
```

Comment:
Since we handle this specially and we might have other clusterclasses in the future, I wonder if it would be better to have SUPPORTED_DISTROS and SUPPORTED_CLUSTERCLASSES

Reply:
Done